estuary/utils/general.py

```diff
@@ -395,3 +395,24 @@ def set_story_labels(requested_node_label, results, reverse=False):
         idx += delta
         node_idx += delta
     return results
+
+
+def create_header(item_type, item_id, item_story_flow, reverse):
+    """
+    Generate a header for the siblings chart.
+
+    :param string item_type: the name of the type of current item
+    :param string item_id: the id of the current item
+    :param dict item_story_flow: has forward and backward relationships of current item
+    :param bool reverse: determines if the results are in the reverse order of the story flow
+    :return: returns the appropriate siblings chart header
+    :rtype: string
+    """
+    if reverse:
+        result = '{0}s triggered by {1} {2}'.format(
+            item_story_flow['backward_label'], item_type, item_id)
+        return result
+    else:
+        result = '{0}s that triggered {1} {2}'.format(
+            item_story_flow['forward_label'], item_type, item_id)
```

Comment:
This should probably be `result = '{0}s that triggered {1} {2}'.format(item_story_flow['backward_label'], item_type, item_id)`

Reply:
Same as above